src/writeData/subscriptions/components/BrokerForm.scss

```
@@ -76,6 +72,12 @@
   &__text {
     color: $cf-grey-95;
   }
+  &__upgrade-text {
+    background: -webkit-linear-gradient(45deg, #34bb55 0%, #00a3ff 100%);
```

```suggestion
    background: -webkit-linear-gradient(45deg, #34bb55 0%, $c-pool 100%);
```

it looks like `#34bb55` is defined in Clockface as `Rainforest` but it doesn't look like there's an scss variable for it.